ipinfo/cmd_login.go

```diff
@@ -58,6 +62,15 @@ func cmdLogin() error {
 		tok = string(tokraw[:])
 	}
 
+	tokenOk, err := checkToken(tok)
+	if err != nil {
+		return err
+	}
+
```

Comment:
Since the line below is checking exactly what `checkToken` returned, this newline can be removed so it's all squeezed together.

Reply:
Hi

I have coded the changes. I would like to merge my branch